packages/nimble-components/src/table/tests/table-sorting.spec.ts

```diff
@@ -252,24 +252,6 @@ describe('Table sorting', () => {
         expect(getRenderedRecordIds()).toEqual(['4', '2', '3', '1']);
     });
 
-    it('performs case sensitive string sort', async () => {
```

Comment:
I removed the tests from this file that were sort-type specific. I thought it would be better to have them in `sort-operations.spec.ts` and leave these tests to be the table behavior around sorting rather than the actual sort algorithm.